docs/common/ex_passmethods.md

```diff
@@ -0,0 +1,115 @@
+\mathcal{F}_1\mathcal{U}(-\varepsilon_1)\mathcal{W}\mathcal{U}(-\varepsilon_2)
+\mathcal{F}_2\mathcal{Y}(\varepsilon_2)` with:
+- {math}`\mathcal{Y}(\varepsilon_1)`: y-axis rotation (Eq. 10.26)
+- {math}`\mathcal{F}_1`: dipole fringe field in the hard-edge limit (Eq. 13.13), [^F2]
+- {math}`\mathcal{U}(-\varepsilon_1)`: entrance wedge (Eq. 12.41)
+- {math}`\mathcal{W}`: bend-kick sequence in cylindrical geometry(Eq. 12.18)
+- {math}`\mathcal{U}(-\varepsilon_2)`: exit wedge
+- {math}`\mathcal{F}_2`: dipole fringe field in the hard-edge limit
+- {math}`\mathcal{Y}(\varepsilon_2)`: y-axis rotation
+
+```{Tip}
+For a pure dipole field, without synchrotron radiation, it is possible to
+integrate the whole magnet in one step, resulting in a much faster transfer. For
+this, use `NumIntSteps=0`.
+```
+
+PolynomB, PolynomA, MaxOrder, NumIntSteps
+: see [StrMPoleSymplectic4Pass](#strmpole)
+
+Length, BendingAngle, EntranceAngle, ExitAngle
+: see [BndMPoleSymplectic4Pass](#bndmpole)
+
+```{Caution}
+`ExactSectorBendPass` show a small discontinuity around origin. Therefore it is
+not recommended to use it for computations based on transfer matrices (linear
+optics). This can be mitigated by increasing the differentiation steps `XYStep`
+and `DPStep` with respect to their default values.
+```
+
+## `ExactRectangularBendPass`
+Exact integration in a bending magnet with Cartesian layout.
+
+This method uses the drift-kick split of the Hamiltonian in the Cartesian
+coordinates of the magnet.
+Following the notations in [^Forest],
+the map corresponds to {math}`\mathcal{Y}(\varepsilon_1)
+\mathcal{F}_1\mathcal{U}(\theta/2-\varepsilon_1)\mathcal{D}
+\mathcal{U}(\theta/2-\varepsilon_2)\mathcal{F}_2\mathcal{Y}(\varepsilon_2)` with:
+- {math}`\mathcal{Y}(\varepsilon_1)`: y-axis rotation (Eq. 10.26)
+- {math}`\mathcal{F}_1`: dipole fringe field in the hard-edge limit (Eq. 13.13), [^F2]
+- {math}`\mathcal{U}(\theta/2-\varepsilon_1)`: entrance wedge (Eq. 12.41)
+- {math}`\mathcal{D}`: drift-kick sequence
+- {math}`\mathcal{U}(\theta/2-\varepsilon_2)`: exit wedge
+- {math}`\mathcal{F}_2`: dipole fringe field in the hard-edge limit
+- {math}`\mathcal{Y}(\varepsilon_2)`: y-axis rotation
+
+For consistency with other passmethods, the `Length` attribute is the length {math}`L`
+of the arc within the magnet, and not the Cartesian length {math}`L_c` of the
+magnet. {math}`L=L_c\frac{\theta/2}{sin(\theta/2)}`
+
+If the magnet field includes quadrupole or higher components, the reference trajectory 
+in the magnet is no more an arc of a circle. A tuning of the `X0ref` attribute is
+necessary to get the correct output angle from the magnet. This can be seen as a
+horizontal translation of the magnet until the correct deviation angle is obtained.
+Similarly, the path lengthening must be adjusted to take into account the
+length of the non-circular trajectory.
+
+This tuning is performed using a dedicated function/method:
+
+- python
```

Comment:
These should be mentioned in the docstrings as well not? Or at least a reference to this shuld be added.

Reply:
The new tuning method is documented if you look at the `Dipole` class.